src/index.js

```diff
@@ -70,7 +70,12 @@ function addPackageData (inputs, path) {
 function computeHash (pepper, paths, seed) {
   const inputs = []
   if (pepper) inputs.push(pepper)
-  if (seed) inputs.push(seed)
+  if (seed) {
```

Comment:
Yea. [Tracing my steps](https://github.com/novemberborn/package-hash/commit/b4db864fed91b40153d3306b1f80af5760154e2b) I originally called `pepper` `salt`, but then added a "salt" but named it "seed", and then renamed the existing `salt` to `pepper`. That freed up `salt` but I never renamed `seed`…